Assets/AppCenter/Plugins/AppCenterSDK/Push/Android/PushInternal.cs

```diff
@@ -35,6 +34,7 @@ private static void PostInitialize()
             AndroidJavaClass unityPlayer = new AndroidJavaClass("com.unity3d.player.UnityPlayer");
             AndroidJavaObject activity = unityPlayer.GetStatic<AndroidJavaObject>("currentActivity");
             instance.Call("onActivityResumed", activity);
+            Push.ReplayPushNotificationsIfWaiting();

```

Comment:
This change @annakocheshkova

Reply:
As I understand, in this case, when notification opens a closed application on Android, we should see an empty notification (no text at all). Before this change in the puppet app notification window I saw text "unity: null" but after this change, there was an empty notification window. As I understand, this is the right behavior.

Reply:
So there is a behavior change then? Then it deserves a changelog entry indeed, going to review it.